components/google_drive/actions/find-folder/find-folder.mjs

```diff
@@ -25,11 +25,17 @@ export default {
       ],
       description: "The name of the folder to search for",
     },
+    includeTrashed: {
+      type: "boolean",
+      label: "Include Trashed",
+      default: true,
+      optional: true,
+      description: "If set to true, returns all matches including items currently in the trash. Defaults to `true`.",
+    }
   },
   async run({ $ }) {
-    const opts = getListFilesOpts(this.drive, {
-      q: `mimeType = '${GOOGLE_DRIVE_FOLDER_MIME_TYPE}' and name contains '${this.nameSearchTerm}'`,
-    });
+    const q = `mimeType = '${GOOGLE_DRIVE_FOLDER_MIME_TYPE}' and name contains '${this.nameSearchTerm}' and trashed=${this.includeTrashed.toString()}`.trim();
+    const opts = getListFilesOpts(this.drive, { q });
```

Comment:
**ESLint**

Line breaks and a comma are expected here.
You can run `npx eslint ./actions/find-folder/find-folder.mjs --fix`, for example, to automatically fix lint issues

```suggestion
    const opts = getListFilesOpts(this.drive, {
      q,
    });
```